src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const orderedStyles = sourceString.split(';');
+
+  const cssObject = {};
+
+  orderedStyles.forEach((style) => {
+    const [key, value] = style.split(':');
+
+    if (key && value) {
+      cssObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using a `forEach` loop to populate `cssObject`, consider using more functional approaches like `reduce` which aligns with the task's requirement to avoid loops and use iteration methods instead.